client/src/Stage.jsx

```diff
@@ -8,17 +8,17 @@ export function Stage() {
   const players = usePlayers();
   const round = useRound();
 
-  if (player.stage.get("submit")) {
-    if (players.length === 1) {
-      return <Loading />;
-    }
+  // if (player.stage.get("submit")) {
```

Comment:
Hey, can you walk me through this change? Do you think we should comment this out for production, or is this just for debugging?